torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -396,13 +399,80 @@ void Reducer::mark_variable_ready_sparse(VariableIndex index) {
     replica.contents = grad;
     // See Note [DDP Communication Hook]
     if (comm_hook_ == nullptr) {
-      replica.contents.div_(process_group_->getSize());
+      replica.contents.div_(divFactor_);
     }
     // The grad is modified in place and needs to be written back.
     return true;
   });
 }
 
+std::vector<std::vector<at::Tensor>> Reducer::getBucketTensors() const {
```

Comment:
existing functions in this file are using snake names (except `rebuildBuckets`), should we stay consistent?